docs/internal_api.yml

```diff
@@ -38,7 +38,76 @@ paths:
             description: Unexpected error.
             schema:
               $ref: '#/definitions/Error'
+ /tenant/{tenant_id}/limits/max_device_count:
```

Comment:
Do we need `/tenant/{tenant_id}/limits` as well? (would return all currently set limits)